java/src/com/ibm/streamsx/topology/generator/spl/OperatorGenerator.java

```diff
@@ -89,6 +90,25 @@ private static void portTypesNote(JSONObject op, StringBuilder sb) {
         }
     }
 
+    private void viewAnnotation(JSONObject op, StringBuilder sb){
+	JSONObject config = (JSONObject)op.get("config");
+	if(config == null)
+	    return;
+	JSONObject viewConfig = (JSONObject)config.get("viewConfig");
```

Comment:
Multiple views can now exist on an operator per port, although this doesn't happen because each time `stream.view()` is invoked, it creates an additional operator. This could be improved in the future to only use one operator for json-ifying the tuples of a stream, regardless of how many times `stream.view()` is invoked.